### PR TITLE
base64 -d mac osx workaround

### PR DESCRIPTION
**Pull request by lotharschulz**
```
base64 --decode
```

### CODE REVIEW

**Comment by lotharschulz** [comment]
👍

**Comment by mikkeloscar** [comment]
Just replace the example with the one that also works on Mac. I think it's more confusing to have two ways instead of a slightly more complicated command that works on both.

**Comment by lotharschulz** [comment]
sure can do

**Comment by lotharschulz** [comment]
👍

File: user-guide/key-encryption.rst [hunk 1]
```diff
@@ -38,12 +38,12 @@ Let's test if all works:
 .. code-block:: bash
 
     # 1. Encrypt and save the binary content to a file:
-    $ aws kms encrypt --key-id $KMS_KEY_ID --plaintext "<here-you-can-paste-your-pwd>"  --query CiphertextBlob --output text | base64 -d > /tmp/encrypted
+    $ aws kms encrypt --key-id $KMS_KEY_ID --plaintext "<here-you-can-paste-your-pwd>"  --query CiphertextBlob --output text | base64 | tr -d '\n' > /tmp/encrypted
```
Comment (mikkeloscar) [inline user-guide/key-encryption.rst]: nit: can you remove the extra spaces in the end? :)

File: user-guide/key-encryption.rst [hunk 2]
```diff
@@ -38,12 +38,12 @@ Let's test if all works:
...
```
Comment (harti2006) [inline user-guide/key-encryption.rst]: what about `base64 --decode` does this work on both systems?

Comment (lotharschulz) [reply user-guide/key-encryption.rst]: `base64 --decode` does not work for me.
The proposed solution `base64 | tr -d '\n'` produces proper text output.

`base64 --decode` produces some binary code I do not expect:
```
??%??b,??1??H?
0|0z0uNM`?He.0?Q??U????0?
```

Comment (harti2006) [reply user-guide/key-encryption.rst]: actually the ciphertext is supposed to be a cryptic random binary string. 
For me testing your change fails with ´InvalidCiphertextException´ (when trying to decrypt the content of tmp/encrypted again)

**Comment by lotharschulz** [comment]
thanks to @harti2006 I could commit https://github.com/zalando-stups/documentation/pull/132/commits/7c8f3f7beea235a23a2c190949749714e9730880 .

**Comment by harti2006** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**